cms/src/components/content/EditContentForm.vue

```diff
@@ -214,7 +232,12 @@ const hasNoPublishErrors = computed(() => {
         return hasParentImage.value;
     }
 
-    return hasOneContentField.value && hasParentImage.value && hasTag.value;
+    return (
+        hasOneContentField.value && hasParentImage.value && hasTag.value && hasNoDraftErrors.value
+    );
+});
+const hasNoDraftErrors = computed(() => {
+    return hasGroup.value;
```

Comment:
I would suggest renaming `hasNoDraftErrors` to `canSaveAsDraft`, and then using that as well to disable the "save as draft" button (like `canPublish` does for publishing).